scripts/west_commands/runners/jlink.py

```diff
@@ -262,14 +262,14 @@ def flash(self, **kwargs):
         # Get the build artifact to flash, preferring .hex over .bin
         if self.hex_name is not None and os.path.isfile(self.hex_name):
             flash_file = self.hex_name
-            flash_cmd = f'loadfile {self.hex_name}'
+            flash_cmd = f'loadfile "{self.hex_name}"'
         elif self.bin_name is not None and os.path.isfile(self.bin_name):
             if self.dt_flash:
                 flash_addr = self.flash_address_from_build_conf(self.build_conf)
             else:
                 flash_addr = 0
             flash_file = self.bin_name
-            flash_cmd = f'loadfile {self.bin_name} 0x{flash_addr:x}'
+            flash_cmd = f'loadfile "{self.bin_name}" {hex(flash_addr)}'
```

Comment:
```suggestion
            flash_cmd = f'loadfile "{self.bin_name}" 0x{flash_addr:x}'
```

Reply:
I've tested both approaches and they work the same so if you think it's worth reverting I will do that.

Reply:
@bbilas Yes please do make the change, it’s cleaner than using hex(). Thanks!

Reply:
Reverted.